moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp

```diff
@@ -49,7 +49,8 @@ static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.planning_sce
 class PlanningSceneInterface::PlanningSceneInterfaceImpl
 {
 public:
-  explicit PlanningSceneInterfaceImpl(const std::string& ns = "", bool wait = true) : node_(new rclcpp::Node(ns))
+  explicit PlanningSceneInterfaceImpl(const std::string& ns = "", bool wait = true)
+    : node_(new rclcpp::Node("planning_scene_interface", ns))
```

Comment:
Does this work the way it should? I would think that this will cause the same duplicate node issues as with the PSM.